framework/src/org/checkerframework/framework/flow/CFAbstractStore.java

```diff
@@ -871,6 +881,10 @@ public S leastUpperBound(S other) {
         return newStore;
     }
 
+    private V upperBound(boolean shouldWiden, V otherVal, V thisVal) {
```

Comment:
For consistency with the `mergeStores` method above, I would use `(V, V, boolean)`. Some javadoc would still be useful.